eng/versioning/version_client.txt

```diff
@@ -121,7 +121,7 @@ com.azure:azure-security-keyvault-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-sdk-template;1.2.1-beta.2;1.2.1-beta.16
 com.azure:azure-spring-data-cosmos;3.14.0;3.15.0-beta.1
 com.azure:azure-spring-data-cosmos-test;3.0.0-beta.1;3.0.0-beta.1
-com.azure:azure-storage-blob;12.14.1;12.15.0-beta.2
+com.azure:azure-storage-blob;12.14.2;12.15.0-beta.2
```

Comment:
This version update usually is done through the autogenerated PR. Was the automated PR to increment the version not created?

Reply:
release was done via release branch.

Reply:
These libraries were released from the release branch. The manual PR that was backported to main missed this library.